recipes/magnum/all/conanfile.py

```diff
@@ -391,6 +434,12 @@ def _patch_sources(self):
                               "EGL::EGL",
                               "egl::egl")
 
+        # Copy FindOpenGLES2/3.cmake to build folder
+        shutil.copy(os.path.join(self._source_subfolder, "modules", "FindOpenGLES2.cmake"), "FindOpenGLES2.cmake")
```

Comment:
shouldn't we remove them? we don't package CMake modules.

Reply:
This is not being packaged. I´m just reusing the [file that the library provides](https://github.com/mosra/magnum/blob/master/modules/FindOpenGLES2.cmake). I really don't know if/how to create a Conan package for it, I don't have enough experience with it... is this a different package? Is the same `opengl` package we have?

Reply:
I think GLES are considered different entities from OpenGL.
why doesn't it find them if they aren't copied?
isn't it enough to adjust `CMAKE_MODULE_PATH`/`CMAKE_PREFIX_PATH`?

Reply:
> why doesn't it find them if they aren't copied?
> isn't it enough to adjust CMAKE_MODULE_PATH/CMAKE_PREFIX_PATH?

In the sources repo, these files are in a directory together with other `FindXXXX` files (https://github.com/mosra/magnum/tree/master/modules), that directory is removed from `CMAKE_MODULE_PATH/CMAKE_PREFIX_PATH` in the recipe. We don't need the rest of them as we are providing those files with Conan.